shenyu-plugin/shenyu-plugin-dubbo/shenyu-plugin-apache-dubbo/src/main/java/org/apache/shenyu/plugin/apache/dubbo/cache/ApacheDubboConfigCache.java

```diff
@@ -26,7 +26,7 @@
 import org.apache.dubbo.config.ConsumerConfig;
 import org.apache.dubbo.config.ReferenceConfig;
 import org.apache.dubbo.config.RegistryConfig;
-import org.apache.dubbo.rpc.model.ApplicationModel;
+import org.apache.dubbo.rpc.model.FrameworkModel;
```

Comment:
if dubbo version is 2.7.X， has this？

Reply:
No, the data fetching place of the upgraded version has changed

Reply:
I tested with 2.7.0 data. Consistent with the current version
dubbo 3.0.9 depends on curator-x-discovery, I see that integration tests cannot pull dependencies

Reply:
What should I do if the user is on version 2.7.x? Can it be compiled over here?

Reply:
shenyu as dubbo client can invoke dubbo server 2.7.x-3.0.9, but user use dubbo2.7.x in shenyu, this can't compile over. 
because of dubbo version upgrade, ConfigManager#getDefaultConsumer was moved to ModuleConfigManager#getDefaultConsumer

Reply:
I think the purpose of `ApplicationModel#getConfigManager()#getDefaultConsumer()` is to get the profile and `ConsumerConfig#refresh()` is also available.After fied compilation can be compatible with dubbo 2.7.X

Reply:
OK. I think I have carefully checked the impact of the Dubbo upgrade. I want to try to join the shenyu community